client/packages.go

```diff
@@ -51,36 +51,36 @@ const (
 	TypeOS        = "os"
 )
 
-// Packages returns the list of packages the system can handle
-func (client *Client) Packages() (map[string]*Package, error) {
-	const errPrefix = "cannot list packages"
+// Snaps returns the list of snaps the system can handle
```

Comment:
the list of snaps the system has the [potential] capabilities to handle. Not sure whether this would be a better wording.